migration/src/m20230131_140613_change_token_account_indexes.rs

```diff
@@ -81,18 +82,10 @@ impl MigrationTrait for Migration {
                 sea_query::Index::create()
                     .name("ta_slot_updated_idx")
                     .index_type(sea_query::IndexType::BTree)
-                    .table(token_accounts::Entity)
+                    .table(TokenAccounts::Table)
                     .to_owned(),
             )
             .await?;
         Ok(())
     }
 }
-
-/// Learn more at https://docs.rs/sea-query#iden
-#[allow(dead_code)]
-#[allow(clippy::upper_case_acronyms)]
-#[derive(Iden)]
-enum Index {
-    BRIN,
-}
```

Comment:
I guess this was just never used?

Reply:
It's mock code that is generated when you setup a new migration. It was never deleted.